src/SDKs/Azure.Base/data-plane/Azure.Base.Tests/HttpClientTransportTests.cs

```diff
@@ -305,5 +305,86 @@ public async Task RequestIdCanBeOverriden()
             var response =  await ExecuteRequest(request, transport);
             Assert.AreEqual(request.RequestId, response.RequestId);
         }
+
+        [Test]
```

Comment:
Should we consider a case to validate that seekable streams support setting position and seeking while non-seekable streams do not?

Reply:
Do you mean a test that verifies that we call into the base stream for those members?

Reply:
In the current implementation, yes, since we delegate that to the contained stream.

Reply:
Done.